tests/mapTest.spec.ts

```diff
@@ -0,0 +1,7 @@
+import { test } from "./fixtures";
+import { expect } from "@playwright/test";
+
+test("Worker marker visible", async ({ page }) => {
+  await page.getByTestId("Tacoma marker").click();
+  await expect(page.getByText("Janet Leverling")).toBeVisible();
```

Comment:
don't use anything besides getByTestId unless it is crucial.
you can get by test id and find the text from the element itself.
Doing it this way might let a bug go through if the text appears anywhere else on the page